<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSerial</class>
 <widget class="QWidget" name="FormSerial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>119</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SerialConfig</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="portName"/>
     </item>
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Desconectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
